packages/trader/src/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -236,12 +237,14 @@ const Chart = props => {
             scrollToEpoch={props.scroll_to_epoch}
             stateChangeListener={props.chartStateChange}
             symbol={props.symbol}
+            allTicks={props.all_ticks}
             topWidgets={ChartTopWidgets}
             isConnectionOpened={props.is_socket_opened}
             isStaticChart={false}
             shouldFetchTradingTimes={!props.end_epoch}
             yAxisMargin={getChartYAxisMargin()}
             anchorChartToLeft={isMobile()}
+            shouldFetchTickHistory={getDurationUnitText(getDurationPeriod(props.contract_info)) === 'seconds'}
```

Comment:
This should be any duration except `tick`, right?

Reply:
yes, but I hunt ticks